core/src/main/java/org/apache/accumulo/core/clientImpl/TabletServerBatchWriter.java

```diff
@@ -280,8 +284,19 @@ public synchronized void addMutation(TableId table, Mutation m)
     mutations.addMutation(table, m);
     totalAdded++;
 
-    if (mutations.getMemoryUsed() >= maxMem / 2) {
+    if (mutations.getMemoryUsed() >= maxMem / MAX_MEMORY_DIVISOR) {
+
       startProcessing();
+
+      if (!somethingFailed)
+        consecutiveFailedProcessCalls.set(0);
+
+      if (consecutiveFailedProcessCalls.get() > 5000)
+        log.error(
```

Comment:
Is this scenario something you've seen occur frequently?

Also, this is going to log every time once it hits this boundary, since the condition isn't reset after logging. It could be very spammy.